nixos/modules/services/misc/gitlab.nix

```diff
@@ -624,6 +624,10 @@ in {
           touch "${cfg.statePath}/db-seeded"
         fi
 
+        # The gitlab:shell:setup regenerates the authorized_keys file so that
+        # the store path to the gitlab-shell in it gets updated
+        ${pkgs.sudo}/bin/sudo -u ${cfg.user} force=yes ${gitlab-rake}/bin/gitlab-rake gitlab:shell:setup RAILS_ENV=production
```

Comment:
`force=yes` is needed here to prevent a user prompt, which requests confirmation for overriding the `authorized_keys` file, to prevent unwanted data loss